pkgs/development/compilers/go/1.15.nix

```diff
@@ -11,8 +11,19 @@ let
 
   inherit (lib) optionals optionalString;
 
+  version = "1.15.8";
```

Comment:
I don't know that we'd want to update the bootstrap for every new patch version, also all the new tarballs will be added to the hydra cache.

We could do `bootstrap_version = ${lib.versions.majorMinor version}` or similar?

Reply:
Even that might be too often, go doesn't require recent features to build the compiler so I assume it would only need to be updated when they make available new bootstrap binaries for an additional architecture.

Reply:
Makes sense, updated to use `lib.versions.majorMinor`.

Reply:
Maybe keep the existing ones at 1.13 and start using 1.16 to bootstrap from then?

Reply:
Take a look at the latest revision and let me know what you think.